src/cargo/util/lints.rs

```diff
@@ -573,6 +601,120 @@ pub fn check_implicit_features(
     Ok(())
 }
 
+const UNKNOWN_LINTS: Lint = Lint {
```

Comment:
You know. I'll request for more doc comments like what `IMPLICIT_FEATURES` has :)